cmd/oras/internal/display/metadata/text/attach.go

```diff
@@ -43,12 +43,12 @@ func NewAttachHandler(printer *output.Printer) metadata.AttachHandler {
 func (ah *AttachHandler) OnAttached(target *option.Target, root ocispec.Descriptor, subject ocispec.Descriptor) {
 	ah.root = root
 	if strings.HasSuffix(target.RawReference, subject.Digest.String()) {
-		ah.subjectRefByDigest = target.AnnotatedReference()
+		ah.subjectRefByDigest = target.GetDisplayString()
```

Comment:
How about `GetDisplayReference`

Reply:
BTW looks like the `subjectRefByDigest` can be changed to `subjectDisplayReference`

Reply:
Done.